website/pages/en/presentations.js

```diff
@@ -0,0 +1,102 @@
+/**
+ * Copyright 2019 Parity Technologies
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+
+ *     http://www.apache.org/licenses/LICENSE-2.0
+
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const React = require("react");
+
+const HomeSplash = require(`${process.cwd()}` + `/core/HomeSplash.js`);
+
+const translate = require("../../server/translate.js").translate;
+
+const Container = require("../../../../react-bootstrap/Container.js");
+const Card = require("../../../../react-bootstrap/Card.js");
+const Button = require("../../../../react-bootstrap/Button.js");
+
+const Col = require("../../../../react-bootstrap/Col.js");
+const Row = require("../../../../react-bootstrap/Row.js");
+
+class Presentations extends React.Component {
+  render() {
+    const { config: siteConfig, language = "" } = this.props;
+    const { baseUrl, docsUrl } = siteConfig;
+    const docsPart = `${docsUrl ? `${docsUrl}/` : ""}`;
+    const langPart = `${language ? `${language}/` : ""}`;
+    const docUrl = doc => `${baseUrl}${docsPart}${langPart}${doc}`;
+    const pageUrl = page => baseUrl + (language ? `${language}/` : "") + page;
+
+    if ((siteConfig.presentations || []).length === 0) {
+      return null;
+    }
+
+    function isFeatured(presentation) {
+      if (presentation.weight < 0) {
```

Comment:
Should it be bigger than 0?

Reply:
The more "weight" it has, the lower to the bottom it is. If weight is negative, it "floats" to the top as featured...

Just an implementation detail

Reply:
This could work, just feel a bit tricky. The suggestion is either we only depend on the order in the presentation list, or only use weight: the bigger one goes to the front.